template.tpl

```diff
@@ -587,33 +615,53 @@ ___SANDBOXED_JS_FOR_SERVER___
 const createRegex = require('createRegex');
 const encodeUriComponent = require('encodeUriComponent');
 const eventData = require('getAllEventData')();
-const getTimestampMillis = require('getTimestampMillis');
 const getType = require('getType');
 const JSON = require('JSON');
 const logToConsole = require('logToConsole');
 const makeNumber = require('makeNumber');
 const makeString = require('makeString');
 const Object = require('Object');
 const sendHttpRequest = require('sendHttpRequest');
-const setResponseBody = require('setResponseBody');
 const setResponseHeader = require('setResponseHeader');
 const setResponseStatus = require('setResponseStatus');
 const sha256Sync = require('sha256Sync');
 const testRegex = require('testRegex');
+const getRequestHeader = require('getRequestHeader');
+const getContainerVersion = require('getContainerVersion');
 
 const ENDPOINT = 'https://' + data.instanceName + '.piwik.pro/ppms.php';
 const LIBRARY_NAME = 'sgtm';
-const LIBRARY_VERSION = '1.0.0';
-const LOG_PREFIX = '[ppms_tag] ';
+const LIBRARY_VERSION = '1.0.1';
+
+const isLoggingEnabled = determinateIsLoggingEnabled();
+const traceId = isLoggingEnabled ? getRequestHeader('trace-id') : undefined;
 
-/** 
- * Logging helper.
+/**
+ * Returns if logging is enabled based on the logType value and the container version.
  *
- * @param {String} message - the message to be logged.
+ * @returns {boolean}
  */
-const log = msg => {
-  logToConsole(LOG_PREFIX + msg);
-};
+function determinateIsLoggingEnabled() {
```

Comment:
Change to `const` like other function declarations and rename to `isLoggingEnabled`:

```
const isLoggingEnabled = () => {
```